Assets/Scripts/Actors/ActorComponents/PlayerActorResources.cs

```diff
@@ -68,9 +68,8 @@ void CheckScroll()
 		if((scrollAmount > WadeUtils.SMALLNUMBER || scrollAmount < -WadeUtils.SMALLNUMBER) & heldResourceTypes.Count > 0)
 		{
 			// Need to do this so >0 rounds up and <0 rounds down
-			int nextIndex = resourceIndex + scrollAmount > 0f ? Mathf.CeilToInt(Mathf.Clamp(scrollAmount, -1f, 1f)) : 
-																Mathf.FloorToInt(Mathf.Clamp(scrollAmount, -1f, 1f));
-			
+			int nextIndex = resourceIndex + scrollAmount > 0f ? Mathf.Clamp( Mathf.RoundToInt( scrollAmount ), -1, 1 ) : 
```

Comment:
The two cases in this ternary operator look the same. Also it is really long and hard to read.